Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -142,13 +138,23 @@ private void Initialize()
                 AddManager(manager.Key, manager.Value);
             }
 
+            #endregion Managers Initialization
+
+            #region Managers Initialization
+
             //Initialize all managers
             foreach (var manager in ActiveProfile.ActiveManagers)
             {
                 manager.Value.Initialize();
             }
 
-            #endregion Managers Initialization
+            #endregion
```

Comment:
add description of the region (like above)

Reply:
OK